hydrolib/core/io/ini/util.py

```diff
@@ -313,7 +313,7 @@ def validate_coordinates(coord_name: str) -> None:
         if str_is_empty_or_none(node_id) and (str_is_empty_or_none(branch_id)):
             if n_coords == 0:
                 raise ValueError(
-                    f"Either {'nodeId, ' if allow_nodeid else ''}branchId (with chainage) or {numfield_name} (with {xfield_name} and {yfield_name}) are required."
```

Comment:
This has become a little hard to read, but we will update this anyway!

Reply:
I agree